collectors/ebpf.plugin/README.md

```diff
@@ -84,27 +162,25 @@ The Agent counts and shows the number of instances where a running program exper
 
 #### Create
 
-This chart shows the number of calls for `vfs_create`. This function is responsible to create files.
+This chart shows the number of calls to `vfs_create`. This function is responsible for creating files.
 
 #### Synchronization
 
-This chart shows the number of calls for `vfs_fsync`. This function is responsible to perform a fsync or fdatasync 
-on a file.
+This chart shows the number of calls to `vfs_fsync`. This function is responsible for calling `fsync(2)` or 
+`fdatasync(2)` on a file. You can see more details on in the Synchronization section.
```

Comment:
```suggestion
`fdatasync(2)` on a file. You can see more details in the Synchronization section.
```

Reply:
Hmm, I didn't ask you to change the first `on`. I asked to remove the second one.